lib/adapter.js

```diff
@@ -1887,28 +1887,40 @@ function Adapter(options) {
                     }
 
                     adapterObjects.getObjectView('system', 'device', {startkey: this.namespace + '.', endkey: this.namespace + '.\u9999', include_docs: true}, (err, _devices) => {
-                        if (_channels && _channels.rows) {
-                            for (let c = _channels.rows.length - 1; c >= 0; c--) {
-                                objects[_channels.rows[c].id] = _channels.rows[c].value;
-                            }
+
```

Comment:
Should we bail early with an error when one of these calls returns an error?

Reply:
Hm that comment wasn't meant to go here... I mean the `err` in the getObjectView callback